speech/google/cloud/speech/sample.py

```diff
@@ -69,6 +71,15 @@ def __init__(self, content=None, source_uri=None,
             raise ValueError('Invalid encoding: %s' % (encoding,))
 
     @property
+    def chunk_size(self):
+        """Chunk size to send over GRPC. ~100ms
+
+        :rtype: int
+        :returns: Optimized chunk size.
+        """
+        return int(self.sample_rate / 10)
```

Comment:
Numerically, there is no benefit, since wiggle room is allowed. But in terms of code readability, using `10.0` makes it explicitly clear that you don't want old-style Python divide / integer division.